### PR TITLE
Lre/test/fix label font modifications test

### PR DESCRIPTION
**Pull request by lredor**
Fix all tests of LabelFontModificationsTest

This PR currently contains a commit to add this test in the gerrit suite to validate the fixes. This last commit should not be merged.

### CODE REVIEW

File: plugins/org.eclipse.sirius.tests.swtbot.support/src/org/eclipse/sirius/tests/swtbot/support/api/editor/SWTBotSiriusGefContextMenu.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                        return true;
+                    }
+                    if (recursive) {
+                        if (invokeMenuInternal(menuItem.getMenu(), matcher, clickInitiated, recursive)) {
+                            return true;
+                        }
+                    }
+                }
+            } finally {
+                if (!bar.isDisposed()) {
```
Comment (lfasani) [inline plugins/org.eclipse.sirius.tests.swtbot.support/src/org/eclipse/sirius/tests/swtbot/support/api/editor/SWTBotSiriusGefContextMenu.java:102]: add a comment that the only change of this whole class is here

**Review by lfasani — CHANGES_REQUESTED** [review]

File: plugins/org.eclipse.sirius.tests.swtbot.support/src/org/eclipse/sirius/tests/swtbot/support/api/editor/SWTBotSiriusGefContextMenu.java (hunk 1 above)
Comment (lredor) [reply plugins/org.eclipse.sirius.tests.swtbot.support/src/org/eclipse/sirius/tests/swtbot/support/api/editor/SWTBotSiriusGefContextMenu.java:102]: Already done on comment line 53

Comment (lredor) [reply plugins/org.eclipse.sirius.tests.swtbot.support/src/org/eclipse/sirius/tests/swtbot/support/api/editor/SWTBotSiriusGefContextMenu.java:102]: I will add another comment...

**Comment by lredor** [comment]
I've added the comment asked by @lfasani and the tests are OK in previous commits (with the commit adding the corresponding tests in "PR suite"). So I soon I had a OK, I will merged.

**Pull request merged**